### ISSUE TITLE
Use function ProfileLookup from codeunit "User Settings Impl."

### ISSUE THREAD

**Issue by AlexCeg** [microsoft/ALAppExtensions#27491]
### Describe the request

The codeunit "User Settings Impl." is internal. Can you make it public or add a call of the function `ProfileLookup` to the codeunit "User Settings", so that I can use the function from my app without cloning the code? Thank you in advance.

### Additional context

We build a functionality where it is neccessary to lookup profiles. The lookup should work the same as in the page "My Settings"

**Comment by JesperSchulz** [comment]
The code you are referring to is open soured here: https://github.com/microsoft/BCApps/blob/main/src/System%20Application/App/User%20Settings/src/UserSettingsImpl.Codeunit.al.

Going forward, you can make the requested changes yourself, if you simply create an issue over there and wait for approval 😊 It's the fastest way to get your changes done!

Now, in this particular case, the function you requested to get exposed, already seems to be exposed?

https://github.com/microsoft/BCApps/blob/21ce62d3ff32b739314487837b1f9ee3bbe555d0/src/System%20Application/App/User%20Settings/src/UserSettings.Codeunit.al#L98

I'm hence closing this issue. If you need further changes, please visit the BCApps repo.

**Comment by AlexCeg** [comment]
Yes you are right, I checked in BC25.0 and not BC25.1. I will use the new function. Thank you. You can close the issue.
